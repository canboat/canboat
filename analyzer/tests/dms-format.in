2024-07-19-23:24:46.790,2,129025,64,255,8,ca,96,5c,16,c8,96,5c,16
