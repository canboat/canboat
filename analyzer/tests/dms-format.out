2024-07-19-23:24:46.790 2  64 255 129025 Position, Rapid Update:  Latitude = 37.5166666; Longitude = 37.5166664
2024-07-19-23:24:46.790 2  64 255 129025 Position, Rapid Update:  Latitude = 37d 31.000 N; Longitude = 37d 31.000 E
2024-07-19-23:24:46.790 2  64 255 129025 Position, Rapid Update:  Latitude = 37d 31' 00.000"N; Longitude = 37d 30' 59.999"E
